CONTRIBUTING.md

```diff
@@ -23,21 +23,13 @@ Licensing is very important to open source projects. It helps ensure the softwar
 
 Some imported or reused components within oneTBB use other licenses, as described in [third-party-programs.txt](https://github.com/oneapi-src/oneTBB/blob/master/third-party-programs.txt). By carefully reviewing potential contributions and enforcing a [Developer Certification of Origin (DCO)](https://developercertificate.org/) for contributed code, we can ensure that the community can develop products with oneTBB without concerns over patent or copyright issues. 
```

Comment:
Looks like this should be rephrased because DCO is mentioned here as well.

Reply:
done!